app/views/sessions/new.html.erb

```diff
@@ -1,5 +1,14 @@
 <%= form_tag session_path do %>
   <h3>Please Login</h3>
+
+  <% flash.each do |key, value| %>
+    <div class="alert alert-<%= key %>">
+      <font color="red">
```

Comment:
font color should be set by CSS based on `alert-X` class; Red probably won't look good on a red background in an `alert-error` div

Reply:
Can you clarify what you mean by an alert-error div and the alert-X class?